pipes/rules/metagenomics.rules

```diff
@@ -50,54 +50,97 @@ method_props = {
     }
 }
 
+taxfiles = [
+                'gi_taxid_nucl.dmp',
+                'gi_taxid_prot.dmp',
+                'names.dmp',
+                'nodes.dmp'
```

Comment:
Ok, and I may ask for your help changing the integration tests for the metagenomics rules.